framework/jobs/src/main/java/org/apache/cloudstack/framework/jobs/dao/AsyncJobDaoImpl.java

```diff
@@ -168,11 +169,11 @@ public void cleanupPseduoJobs(long msid) {
     }
 
     @Override
-    public List<AsyncJobVO> getExpiredCompletedJobs(Date cutTime, int limit) {
-        SearchCriteria<AsyncJobVO> sc = expiringCompletedAsyncJobSearch.create();
-        sc.setParameters("created", cutTime);
+    public List<AsyncJobVO> getExpiredCompletedJobs(final Date cutTime, final int limit) {
+        final SearchCriteria<AsyncJobVO> sc = expiringCompletedAsyncJobSearch.create();
```

Comment:
We'll need to discuss this bit over our syncup call when you're back.

Reply:
Suggestion: during the upgrade 4.11.1->4.12.0 -> set all the removed field of the async_jobs -> `NOW()`?

Reply:
Update added to schema-41110to41200.sql

Reply:
Update added.